paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/embedded/EmbeddedPlaygroundActivity.kt

```diff
@@ -119,6 +120,9 @@ internal class EmbeddedPlaygroundActivity :
             .confirmCustomPaymentMethodCallback(this)
             .externalPaymentMethodConfirmHandler(this)
             .analyticEventCallback(this)
+            .rowSelectionBehavior(
+                playgroundSettings[EmbeddedRowSelectionBehaviorSettingsDefinition].value.rowSelectionBehavior
```

Comment:
Hmm, this is tricky. Changing the setting won't actually update this. 

I think we either need to update this when clicking reload, or we need to automatically recreate the activity when changing the setting.

Reply:
Ohh right! That works!